src/frontend/Miljø.ts

```diff
@@ -1,27 +1,37 @@
 interface MiljøProps {
     loginService: string;
     visInnsendingsknapp: boolean;
+    mellomlagerUrl: string;
+    modellVersjon: number;
```

Comment:
Modellversjon brukes til å sjekke om modellen har endret seg. Hvis det har blitt gjort endringer, så må bruker starte ny søknad for å få oppdatert modell.